tools/tap-phantom

```diff
@@ -33,6 +33,13 @@ if (phantom.args.length == 1) {
 } else if (phantom.args.length == 2) {
     server = sys.args[1];
     url = sys.args[2];
+} else if (phantom.args.length == 3) {
```

Comment:
Could you add a comment as to exactly which automake behavior we're working around here? Just for reference.